.devcontainer/devcontainer.json

```diff
@@ -11,7 +11,11 @@
 		"ghcr.io/devcontainers/features/python:1": {
 			"installTools": true,
 			"installJupyterlab": true,
-			"version": "latest"
+			"version": "3.11"
+		},
+		"ghcr.io/devcontainers/features/azure-cli:1": {
+			"version": "latest",
+			"extensions": "quantum"
```

Comment:
Why is this here? The azure-cli is pretty huge. Does anything depend on this? Or just for the convenience of anyone spinning up a container who wants to run `az login`?

Reply:
Yeah, for login and job monitoring. Based on local testing it makes the container around 800MB bigger, maybe a bit more. I thought it would be useful to have by default, but if we don't think it's central enough to our workflow I can take it out.

Reply:
maybe it would be worth commenting it out so folks can see this is how you can add it to a devcontainer definition?

Reply:
It's fine. I guess you need it (currently) for auth in Python.